checks/container.go

```diff
@@ -97,6 +93,8 @@ func fmtContainers(
 		lastByID[c.ID] = c
 	}
 
+	containerServiceTags := kubernetes.GetContainerServiceTags()
```

Comment:
nit: generally I prefer names that make the key/val clear so I may not have to look at the function API. Something like `serviceTagsByID` might be nice.